README.md

```diff
@@ -1,18 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> One paragraph statement about the project.
+> First Module project to master the tools necessary to complete industry standard development projects using best-practices and tools.
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML / CSS
 
 ## Live Demo
```

Comment:
- [ ] Please consider deleting all unnecessary headings in your `readme` file such as 

- Live Demo
- Run tests

This way your README would look more professional